packages/opensrp-reducer-factory/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@opensrp/reducer-factory",
+  "version": "0.0.3",
```

Comment:
lets update this to 0.0.1

Reply:
@ciremusyoka But this is the same published package, it's not a new package

Reply:
Right @kelvin-muchiri i understand.